tasks/main.yml

```diff
@@ -68,7 +68,9 @@
   seboolean:
     name: "{{ item.name }}"
     state: "{{ item.state }}"
-    persistent: "{{ item.persistent | default('no') }}"
+    persistent: "{{ item.persistent |
+                  default(ansible_selinux.status == 'disabled') }}"
```

Comment:
This changes the data type, and the data type will be different - the result of `ansible_selinux.status == 'disabled'` will be `true` or `false` of type `bool` - can the `seboolean` module take either a string `"yes"`/`"no"` value, or a boolean `true`/`false` value?

Reply:
Does it matter? `persistent` option is `bool` and according to https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_variables.html#boolean-variables `bool` could be both `true` or `'yes'`

Reply:
ok - looks like https://github.com/ansible-collections/ansible.posix/blob/main/plugins/modules/seboolean.py#L278 does convert bool-ish strings to `bool`